Adds SCG banners

comma opened the pull request:
Wooden ones aren't mapped anywhere yet.
Main point is steel deployable ones. They need pathfinder access (who gets 2 of them), and store who and when stuck them in, wow!

<!-- 
Do not forget to add a changelog when you have made admin/player facing changes that can alter gameplay.
Examples which require a changelog entry include:
* Adding/removing objects that players may interact with, or the way they function.
* Adding/removing/altering admin tools.
* Changing the map.

Examples were changelog entries are optional/not typically required:
* Cosmetic changes such as descriptions, sound effects, etc.
* Optimizations and other changes to underlying systems which do not affect gameplay.
* Minor bug fixes.

You find a README and example file in .\html\changelogs\ for further instructions.
-->

MistakeNot4892 commented on maps/torch/items/solbanner.dm:
```diff
@@ -0,0 +1,54 @@
+	plane = ABOVE_HUMAN_PLANE
+
+/obj/structure/solbanner/exo
+	name = "\improper exoplanet SCG banner"
```
Doesn't need `\improper`.

MistakeNot4892 commented on maps/torch/items/solbanner.dm:
```diff
@@ -0,0 +1,54 @@
+		to_chat(user, "<span class='notice'>[plantedby]</span>")
+
+/obj/item/solbanner
+	name = "\improper SCG banner capsule"
```
As above.

MistakeNot4892 replied:
I'm not very bright.

MistakeNot4892 commented on maps/torch/items/solbanner.dm:
```diff
@@ -0,0 +1,54 @@
+		to_chat(user, "<span class='warning'>\The [src] does not recognize your authority!</span>")
+		return
+	if(user.unEquip(src))
+		dropInto(user.loc)
```
You remove the item from inventory before you bother to check whether or not it's suitable for the resulting loc.

babydoll commented on maps/torch/torch_define.dm:
```diff
@@ -37,10 +37,10 @@
 	default_law_type = /datum/ai_laws/solgov
 	use_overmap = 1
-	num_exoplanets = 1
+	num_exoplanets = 0
```
Intentional?

comma replied:
woops

comma committed:
Adds SCG banners

Wooden ones aren't mapped anywhere yet.
Main point is steel deployable ones. They need pathfinder access (who gets 2 of them), and store who and when stuck them in, wow!

Zuhayr commented on maps/torch/items/solbanner.dm:
```diff
@@ -0,0 +1,54 @@
+		to_chat(user, "<span class='notice'>[plantedby]</span>")
+
+/obj/item/solbanner
+	name = "SCG banner capsule"
```
Needs `\improper`.

MistakeNot4892 commented on maps/torch/items/solbanner.dm:
```diff
@@ -0,0 +1,54 @@
+		if(istype(ID.military_rank))
+			dudename = "[ID.military_rank.name] [dudename]"
+		E.plantedby = "Planted on [stationdate2text()] by [dudename], [user.get_assignment()] of [GLOB.using_map.full_name]."
+		to_chat(user,"<span class='notice'>You successfully claim this world with \the [E]!</span>")
```
Should probably be a `visible_message` so everyone else can witness.

MistakeNot4892 reviewed (dismissed):
Other than visible message request this looks good.
